app/src/main/res/layout-land/ongoing_story_card.xml

```diff
@@ -53,37 +53,33 @@
         app:layout_constraintTop_toBottomOf="@id/lesson_thumbnail">
 
         <TextView
+          style="@style/Subtitle2"
           android:id="@+id/story_name_text_view"
           android:layout_width="match_parent"
-          android:layout_height="wrap_content"
           android:layout_marginStart="8dp"
           android:layout_marginTop="8dp"
           android:layout_marginEnd="8dp"
           android:ellipsize="end"
-          android:fontFamily="sans-serif"
           android:maxLines="1"
           android:text="@{viewModel.ongoingStory.storyName}"
           android:textColor="@color/oppiaPrimaryText"
-          android:textSize="16sp"
           app:layout_constraintEnd_toEndOf="parent"
           app:layout_constraintStart_toStartOf="parent"
           app:layout_constraintTop_toTopOf="parent" />
 
         <TextView
+          style="@style/Caption"
           android:id="@+id/topic_name_text_view"
           android:layout_width="match_parent"
-          android:layout_height="wrap_content"
           android:layout_marginStart="8dp"
           android:layout_marginTop="4dp"
           android:layout_marginEnd="8dp"
           android:layout_marginBottom="12dp"
           android:ellipsize="end"
-          android:fontFamily="sans-serif-light"
```

Comment:
This font is needed here because `Caption` uses `sans-serif-medium` but this view needs this font.

Reply:
@rt4914, I have added the font